zpages/rpcz.go

```diff
@@ -58,10 +58,13 @@ var (
 )
 
 func init() {
-	for view := range viewType {
-		if err := view.Subscribe(); err != nil {
-			log.Printf("error subscribing to view %q: %v", view.Name(), err)
-		}
+	views := make([]*view.View, 0, len(viewType))
+	for k := range viewType {
```

Comment:
```go
for view := range viewType
```
as it was before is nicer IMHO than `k`

Reply:
agreed with the name, the reason I changed it was (a) `Subscribe` is deprecated and (b) `view` variable shadows `view` package but I agree `k` is not right